pyglmnet/pyglmnet.py

```diff
@@ -2,10 +2,11 @@
 
 import logging
 from copy import deepcopy
-
+import collections
```

Comment:
can you remove all changes that pertain to estimator checks or `GLMCV`? it can be very confusing for folks looking at the history of a feature. we can redo these changes later.